src/main/java/woohakdong/server/api/service/clubMember/ClubMemberService.java

```diff
@@ -0,0 +1,81 @@
+package woohakdong.server.api.service.clubMember;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import woohakdong.server.api.controller.clubMember.dto.ClubMemberInfoResponse;
+import woohakdong.server.common.exception.CustomException;
+import woohakdong.server.domain.club.ClubRepository;
+import woohakdong.server.domain.clubmember.ClubMember;
+import woohakdong.server.domain.clubmember.ClubMemberRepository;
+import woohakdong.server.domain.member.Member;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static woohakdong.server.common.exception.CustomErrorInfo.CLUB_NOT_FOUND;
+
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class ClubMemberService {
+
+    private final ClubMemberRepository clubMemberRepository;
+
+    public List<ClubMemberInfoResponse> getMembers(Long clubId) {
+        LocalDate assignedTerm = getAssignedTerm();
+
+        // clubId와 assignedTerm으로 클럽 멤버 조회
+        List<ClubMember> clubMembers = clubMemberRepository.findByClubIdAndAssignedTerm(clubId, assignedTerm)
+                .orElseThrow(() -> new CustomException(CLUB_NOT_FOUND));
```

Comment:
```suggestion
        List<ClubMember> clubMembers = clubMemberRepository.findByClubIdAndAssignedTerm(clubId, assignedTerm);
```

이 서비스 메소드는 현재 클럽 멤버를 보여주는 것으로 보여지는데,
제가 생각한 로직이 맞다면 여기는 클럽 멤버가 없더라도 예외 처리가 아니라 빈 배열로 내려가는게 맞을 것 같습니다. 레포지토리 단에서 Optional 제거하는 쪽으로 수정해야할 것 같은데 혹시 다른 의도가 있나요?